src/audio/drc/drc_math_hifi3.c

```diff
@@ -13,7 +13,7 @@
 #include <xtensa/tie/xt_hifi3.h>
 
 #define ONE_OVER_SQRT2_Q30 759250112 /* Q_CONVERT_FLOAT(0.70710678118654752f, 30); 1/sqrt(2) */
-#define LOG10_FUNC_A5_Q26 75959200 /* Q_CONVERT_FLOAT(1.131880283355712890625f, 26) */ 
```

Comment:
Oops, forgot to update commit text. I noticed afterwards that in 1000 step check of inverse there was a few  differing with 1 least significant bit. I think I should add a cmocka unit test for this function.

Reply:
ack - cmocka would be good here.